variables.tf

```diff
@@ -197,12 +197,12 @@ variable "lifecycle_rules" {
 
 variable "cors" {
   description = "Map of maps of mixed type attributes for CORS values. See appropriate attribute types here: https://www.terraform.io/docs/providers/google/r/storage_bucket.html#cors"
-  type        = any
+  type        = map(any)
```

Comment:
I technically agree that this would make more sense here. I was just following the description which clarifies this being a "map of maps". The only downside might be a complete reconfiguration of all CORS definition in case one list element gets removed. @morgante WDYT?

Reply:
I think ideally we accept a set. I don't think reordering should matter much since they're all blocks of the same resource.